src/Apps/Order/Routes/Payment/index.tsx

```diff
@@ -41,7 +41,7 @@ export class PaymentRoute extends Component<PaymentProps> {
               </>
             }
             Sidebar={
-              <Summary mediator={this.props.mediator}>
+              <Summary mediator={this.props.mediator} order={order as any}>
```

Comment:
Apparently the need for this `as any` will be fixed by an upcoming @alloy PR but for now we still need it so the typing for the fragment container is correct.